README.md

```diff
@@ -1011,7 +1014,30 @@ Enable/Disable autostate on the SVI interface. Valid values are 'true',
 Enable/Disable management on the SVI interface. Valid values are 'true', 'false', and 'default'.
 
 --
+### Type: cisco_interface_service
 
+Manages a Cisco Network Interface Service.
+
+#### Parameters
+
+##### Basic interface service config attributes
+
+###### `ensure`
+Determine whether the interface config should be present or not. Valid values are 'present' and 'absent'.
+
+###### `interface`
+Name of the interface where the service resides. Valid value is a string.
+
+###### `sid`
+The Service ID number. Valid value is an Integer.
+
+###### `encapsulation_profile_vni`
+The VNI Encapsulation Profile Name. Valid values are String or the keyword 'default'
```

Comment:
Do we have a provider planned for configuring the VNI encapsulation profile itself?